lighthouse-cli/bin.ts

```diff
@@ -16,14 +16,12 @@
  */
 'use strict';
 
-const _SIGINT = 'SIGINT';
-const _SIGINT_EXIT_CODE = 130;
 const _RUNTIME_ERROR_CODE = 1;
 const _PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 const getFilenamePrefix = require('../lighthouse-core/lib/file-namer.js').getFilenamePrefix;
-import {ChromeLauncher} from '../chrome-launcher/chrome-launcher';
+import {launch, LaunchedChrome} from '../chrome-launcher/chrome-launcher';
```

Comment:
ya agree with eric. I like that eventually the ChromeLauncher class could also be exported and someone could continue use our convenience .launch() or go deeper and instantiate the class themselves.. but for now the `ChromeLauncher` class isn't exported.

for now i think it'd be more clear, especially for the dev team, to rename the `ChromeLauncher` class, its name does feel rather externally facing. ;)